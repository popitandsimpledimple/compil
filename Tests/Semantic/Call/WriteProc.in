procedure a();
begin
end;

begin
    writeln(a())
end.