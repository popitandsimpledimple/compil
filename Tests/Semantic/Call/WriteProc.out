(6,13) Can't read or write variables of this type